lib/import-export-visitor.js

```diff
@@ -70,13 +70,14 @@ module.exports = class ImportExportVisitor extends Visitor {
       }
 
       bodyInfo.body = null;
-      bodyInfo.parent = null;
       bodyInfo.hoistedExportsMap = null;
       bodyInfo.hoistedExportsString = null;
       bodyInfo.hoistedImportsString = null;
       bodyInfo.insertCharIndex = null;
       bodyInfo.insertNodeIndex = null;
       bodyInfo.needToAddUseStrictDirective = null;
+      bodyInfo.parent._bodyInfoByName = null;
```

Comment:
Cool. Might be worth actually deleting the property, since it's attached to a shared AST node, but nulling it is probably fine.